integration_tests/seeds/docs/test_fct_undocumented_sources.csv

```diff
@@ -0,0 +1,2 @@
+resource_name
+source_2.table_3
```

Comment:
@epapineau in terms of what we want to display, would it make sense to either strip this string to `source_2` rather than including the table name too? we may have that as a column in the resources table already! Not blocking, but may clarify what the user needs to do to clear this check

Reply:
💯